controllers/approve-places-admin.js

```diff
@@ -32,7 +34,10 @@ exports.getApprovePlaces = async (req, res) => {
 exports.approvePlace = async (req, res) => {
     try {
         const { id } = await req.body;
-        await places.findByIdAndUpdate({ _id: id }, { isModerated: true });
+        const { authorization } = req.headers;
+        const { userId } = jwt.verify(authorization, adminJwtConf.secret);
+
+        await places.findByIdAndUpdate({ _id: id }, { isModerated: true, moderateBy: userId });
```

Comment:
according to mvc this logic must be written in places model

Reply:
Done

Reply:
Done